File: c/iniciante/BEE1006.c
#include <stdio.h>

int main() {
  double a, b, c;

  scanf("%lf %lf %lf", &a, &b, &c);

  a = a * 2.0;
  b = b * 3.0;
  c = c * 5.0;
  
  double media = (a + b + c) / 10.0;

  printf(("MEDIA = %.1lf\n"), media);

  return 0;
}

File: go/iniciante/BEE1006.go
package main

import (
	"fmt"
)

func main() {
	var a float64
	fmt.Scan(&a)
	
	var b float64
	fmt.Scan(&b)
	
	var c float64
	fmt.Scan(&c)

	var media float64 = ((a * 2.0) + (b * 3.0) + (c * 5.0)) / 10.0

	fmt.Printf("MEDIA = %.1f\n", media)
}